defaults/solo.json

```
@@ -42,12 +42,26 @@
       "value": true
     },
     {
+        "for": "player",
+        "type": "text",
+        "label": "Playlist",
+        "code": "playlist",
+        "value": "show"
```

I think default should be "hide"